libs/community/langchain_community/chat_models/hunyuan.py

```diff
@@ -73,10 +78,10 @@ def _create_chat_result(response: Mapping[str, Any]) -> ChatResult:
     generations = []
     for choice in response["Choices"]:
         message = _convert_dict_to_message(choice["Message"])
+        message.id = response.get("Id", "")
         generations.append(ChatGeneration(message=message))
 
-    token_usage = response["Usage"]
-    llm_output = {"token_usage": token_usage}
+    llm_output = {"token_usage": response.get("Usage", "")}
```

Comment:
is empty string the correct type for token usage?

Reply:
ok，Changes have been rolled back